### PR TITLE
Update iso rebuild to be EFI bootable

### LINKED ISSUE #7113: ISO Installer GRUB path
## Error at end of installer GRUB Fails

When installing using the ISO, if device is not /dev/sda grub-install fails.

image

## Using devices other then /dev/sda fails

Using XCP-NG as hypervisor, devices are enumerated as /dev/xvd[a-z] 

image

### LINKED ISSUE #7599: ISO doesn't boot in UEFI
**Describe the bug**
The ISO doesn't boot in a UEFI VM (and presumably physical UEFI too)

**To Reproduce**
1. Configure VM in UEFI mode, try to boot PF ISO
2. ISO fails to boot

**Expected behavior**
PacketFence installs Debian in UEFI mode

**Desktop (please complete the following information):**
 - OS: VMware ESXi
 - Version: 7.0

**Additional context**
This also affects Hyper-V Gen 2 VMs

### PR DESCRIPTION
**Pull request by trs80**
# Description
This updates the ISO rebuild process to include the bits for EFI booting

# Impacts
Test booting from UEFI on physical, VMware and/or Hyper-V Gen 2 VM

# Issue
fixes #7599 

# Delete branch after merge
YES

# Checklist
- [n/a] Document the feature
- [n/a] Add unit tests
- [n/a] Add acceptance tests (TestLink)

# NEWS file entries

## Enhancements
* ISO supports UEFI booting

### CODE REVIEW

**Comment by nqb** [comment]
Hello @trs80,

Are we sure this change is compatible with non-UEFI boots ?

**Comment by trs80** [comment]
I'm back in the office, and tested it with both BIOS and UEFI booting using GNOME Boxes (which uses KVM), they are both fine. I can test it in VMware if you'd like. It just adds extra information so that UEFI works again, the original Debian ISO you extract from uses the same commands and is bootable by both - you can see the full commandline in `.disk/mkisofs` in the ISO image, see https://wiki.debian.org/RepackBootableISO for more information.

File: ci/debian-installer/create-debian-installer.sh [hunk 1]
```diff
@@ -41,6 +41,6 @@ find -follow -type f ! -name md5sum.txt -print0 | xargs -0 md5sum > md5sum.txt |
 chmod -w md5sum.txt
 cd ..
 
-genisoimage -r -J -b isolinux/isolinux.bin -c isolinux/boot.cat -no-emul-boot -boot-load-size 4 -boot-info-table -o $ISO_OUT isofiles
+xorriso -as mkisofs -r -J -joliet-long -b isolinux/isolinux.bin -c isolinux/boot.cat -boot-load-size 4 -boot-info-table -no-emul-boot -o $ISO_OUT -eltorito-alt-boot -e boot/grub/efi.img -no-emul-boot -isohybrid-gpt-basdat -isohybrid-apm-hfsplus -V "Packetfence $PF_RELEASE" isofiles
```
Comment (nqb) [inline ci/debian-installer/create-debian-installer.sh:44]: Looks like `-no-emul-boot` is called twice.

**Review by nqb — CHANGES_REQUESTED** [review]

File: ci/debian-installer/create-debian-installer.sh (hunk 1 above)
Comment (nqb) [reply ci/debian-installer/create-debian-installer.sh:44]: I removed extra `-no-emul-boot` and I'm testing changes on my own branch.

**Comment by nqb** [comment]
Closed in favor of #7724

**Pull request closed without merging**